src/gui.rs

```diff
@@ -335,9 +96,9 @@ pub type Float = f32;
 /// The current choice of `u16` limits the vertex arrays to a length of 65535.
 pub type Index = u16;
 
-/// A vertex of a 3D [`Primitive3`].
+/// A vertex of a 3D [`Primitive`].
```

Comment:
Maybe `a 3D` is not actual because now `Vertex` struct responsible be a Primitive for 3D and 2D contexts.

But I don't understand how will `position` field works if 2D context because it have 3 length.